### PR TITLE
DHFPROD-2415: Committed/Failed count problems

### PR DESCRIPTION
**Pull request by hao1st**
https://project.marklogic.com/jira/browse/DHFPROD-2415

### CODE REVIEW

File: marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobStatus.java [hunk 1]
```diff
@@ -35,4 +37,20 @@ public String toString() {
     public static final String FAILED_PREFIX = "failed step ";
     public static final String CANCELED_PREFIX = "canceled step ";
 
+    public static boolean jobIsDone(String jobStatus) {
+        return JobStatus.FINISHED.toString().equals(jobStatus) ||
```
[the remark this answers is not in the document]
Comment (aebadirad) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobStatus.java]: Yeah, that seems more consistent

[the remark this answers is not in the document]
Comment (hao1st) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobStatus.java]: good suggestion! thanks.

**Comment by aebadirad** [comment]
We picked up a pair of test failures, @hao1st , could you fix those?

**Comment by hao1st** [comment]
OK. checking

**Pull request closed without merging**